doc/rbd/rbd-snapshot.rst

```diff
@@ -65,76 +71,96 @@ snapshots using the ``rbd`` command.
 Create Snapshot
 ---------------
 
-To create a snapshot with ``rbd``, specify the ``snap create`` option, the pool
-name and the image name.  ::
+To create a snapshot use the ``rbd snap create`` command and specify the pool
+name and the image name:
```

Comment:
`rbd snap create` won't work without the desired snap name: "the pool name, the image name, and the snap name".

Reply:
Done.